### ISSUE TITLE
Selective input of priors

### ISSUE THREAD

**Issue by steveyang90** [uber/orbit#86]
Allow users to use dictionary for `regressor_beta_prior` and `regressor_sigma_prior` args to selectively input priors. 

For example, suppose we have `feature1` through `feature5`, but only want custom priors for `feature1` and `feature4`. Users could set the following args:

```python
lgt = LGT(
    regressor_cols=['feature1', 'feature2', 'feature3', 'feature4', 'feature5'],
    regressor_beta_prior={'feature1': 100, 'feature4': 1000} ,
    regressor_sigma_prior={'feature1': 20, 'feature4': 40}
)
```

**Comment by edwinnglabs** [comment]
Something slightly related to this: I notice our input args in the LGT and DLT final model are masked.  Is there anyway to make it visible in the help manual?

**Comment by steveyang90** [comment]
@edwinnglabs I think that is a separate issue. If I understand correctly, what you're looking to do is have docstrings from parent classes be inherited. I'm not sure what the best way to do this is, but let's open a separate issue on this and explore

**Comment by edwinnglabs** [comment]
Yeah. Let's make a separate issue.  Thanks for the clarification.

**Comment by skyesheng** [comment]
> Allow users to use dictionary for `regressor_beta_prior` and `regressor_sigma_prior` args to selectively input priors.
> 
> For example, suppose we have `feature1` through `feature5`, but only want custom priors for `feature1` and `feature4`. Users could set the following args:
> 
> ```python
> lgt = LGT(
>     regressor_cols=['feature1', 'feature2', 'feature3', 'feature4', 'feature5'],
>     regressor_beta_prior={'feature1': 100, 'feature4': 1000} ,
>     regressor_sigma_prior={'feature1': 20, 'feature4': 40}
> )
> ```

Backing up this idea. Would love to have selected defined priors and leave others with only specified signs.
